### ISSUE TITLE
Readable ordering for sidebar layouts

### ISSUE THREAD

**Issue by awead** [psu-stewardship/scholarsphere-4#480]
Pages that utilize a sidebar/content-on-the-right layouts, need to "read" in an order that reflects the layout of the page. The main content that is present on the right-hand side of the page should be read by a screen reader before the content in the side bar. Currently, the reverse happens, and a screen reader reads the sidebar content before reading the main content.

## WDLL
* swap sidebar and main areas in the html

## Example

The circled `div.container-fluid` should come before the `aside.sidebar` in the HTML while maintaining the same layout.

![Screen Shot 2020-08-18 at 9 31 08 AM](https://user-images.githubusercontent.com/312085/90519883-a02cde00-e136-11ea-89ac-b65746dd8faa.png)
